WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/FIle List/ProductDownloadListViewController.swift

```diff
@@ -121,15 +121,17 @@ extension ProductDownloadListViewController {
     @objc private func doneButtonTapped() {
         // TODO: - add analytics
         viewModel.completeUpdating(onCompletion: onCompletion)
+        navigationController?.popViewController(animated: true)
     }
 
     @objc private func downloadSettingsButtonTapped() {
```

Comment:
I think it's clear enought that we are in downloads.
```suggestion
    @objc private func settingsButtonTapped() {
```

Reply:
updated